cygnus-ngsi/api/v1/index.html

```diff
@@ -0,0 +1,15 @@
       \"reference\": \"http://<reference_host>:<reference_port>\",\n           \"duration\": \"P1M\",\n           \"notifyConditions\": [\n                   {\n                       \"type\": \"ONCHANGE\",\n                      \"condValues\": []\n                   }\n               ],\n           \"throttling\": \"PT5S\"\n           },\n           \"endpoint\":{\n                   \"host\":\"<endpoint_host>\",\n                   \"port\":\"<endpoint_port>\",\n                   \"ssl\":\"false\",\n                   \"xauthtoken\":\"234123123123123\"\n           }\n       }\n\n+ Response 200 (application/json)\n\n        {\n            \"success\":\"true\",\n            \"result\" : {\n                    {\n                      \"subscribeResponse\":{\n                                  \"duration\":\"P1M\",\n                                  \"throttling\":\"PT5S\",\n                                  \"subscriptionId\":\"56f9081c3c6fb7e9d2a912a0\"\n                        }\n                    }\n              }\n        }\n\n+ Response 400 (application/json)\n\n        {\n            \"success\":\"true\",\n            \"result\" : {\n                    \"subscribeError\": {\n                                \"errorCode\": {\n                                        \"code\":\"400\",\n                                        \"reasonPhrase\":\"Bad Request\",\n                                        \"details\":\"JSON Parse Error: unknown field: \\/extraField\"\n                                }\n                    }\n            }\n        }\n\n+ Response 400 (application/json)\n\n        {\n            \"success\":\"false\",\n            \"error\":\"Invalid subscription, field \'duration\' is empty\"\n        }\n\n+ Response 400 (application/json)\n\n        {\n            \"success\":\"false\",\n            \"error\":\"Invalid subscription, field \'notifyConditions\' is missing\"\n        }\n\n## Group Admin API\n\n### Gets the log4j configuration [GET /admin/log]\n\n+ Request\n\n       GET http://<cygnus_host>:<management_port>/admin/log\n\n+ Response 200 (application/json)\n\n       {\n           \"log4j\": {\n           \"appenders\": [\n               {\n                   \"layout\": \"...\",\n                   \"name\": \"...\"\n               }\n            ],\n            \"level\": \"...\"\n           },\n           \"success\": \"true\"\n       }\n\n+ Response 500 (application/json)\n\n       {\n           \"error\": \"...\"\n       }\n\n### Update the logging level of Cygnus [PUT /admin/log]\n\n+ Parameters\n\n        + loglevel: Must be DEBUG, INFO, WARN, ERROR o FATAL (String) - A valid log level.\n\n+ Request\n\n       PUT http://<cygnus_host>:<management_port>/admin/log?level=<log_level>\n\n+ Response 200 (text/plain)\n\n       OK\n\n+ Response 400 (application/json)\n\n       {\n           \"error\":\"Invalid log level\"\n       }\n\n+ Response 400 (application/json)\n\n       {\n           \"error\":\"\"\n       }\n"
```

Comment:
Why `fiware-ngsi2.apiblueprint.org`? It seems to be something highly coupled to Orion NGSI, not Cygnus.